src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,16 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { User } from '../../types/User';
+
+type Props = {
+  user: User
```

Comment:
Consider renaming `user` prop to `userInfo` for better readability and consistency. 🤔